src/type/definition.d.ts

```diff
@@ -292,6 +292,7 @@ export class GraphQLScalarType {
   extensions: Maybe<Readonly<Record<string, any>>>;
   astNode: Maybe<ScalarTypeDefinitionNode>;
   extensionASTNodes: Maybe<ReadonlyArray<ScalarTypeExtensionNode>>;
+  specifiedByUrl?: Maybe<string>;
```

Comment:
Can you please move it between `description` and `serialize` in all types.

Reply:
Fixed in 4c7d339d and bd6d4968.  There were some instances where `specifiedByUrl` and `serialize` were both present , but not `description`.  In those cases I moved `specifiedByUrl` immediately before `serialize`.